.github/workflows/wheels-windows.yml

```diff
@@ -0,0 +1,43 @@
+name: wheels-windows
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CC: cl.exe
+  CXX: cl.exe
+
+jobs:
+  tests:
+    name: Python Wheels Windows
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v2.1.1
+        with:
+          submodules: recursive
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - uses: ilammy/msvc-dev-cmd@v1.2.0
+
+      - name: Build wheels
+        env:
+          CIBW_SKIP: pp* cp27* *-win32
+          CIBW_BEFORE_BUILD: pip install scikit-build cython cmake
+          CIBW_TEST_REQUIRES: pytest numpy
+          CIBW_TEST_COMMAND: pytest {project}/tests
+        run: |
+          pip install --upgrade cibuildwheel
+          python -m cibuildwheel --output-dir wheelhouse
+
+      # - name: Upload artifacts to github
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     name: wheels
+      #     path: ./wheelhouse
```

Comment:
What is the purpose of building these without uploading them somewhere?

Reply:
I understand the point here is to test everything and then to enable uploading after the other parts are done.

Reply:
Yeah. It seemed a bit wasteful to have github store the artifacts after every push. In an upcoming step, we'll figure out how we want to gate the uploads and push to PyPI to only happen for releases.